interviews/backend-takehome.md

```diff
@@ -0,0 +1,43 @@
+# Backend Takehome Grading
+
+## Section Scores
+
+### Solution Completion
+
+All requirements are met as outlined by the assignment's requirements.
+
+- 💖 4 - Perfect
+- 👍 3 - One minor bug with flow using expected inputs, or a couple bugs with validation of corner cases
+- 👎 2 - Two or more missed requirements
+- 🛑 1 - Several requirements missed; compile errors that are not trivial to resolve
+
+### Code Quality
+
+Code is generally well-organized and easy to understand.
+
+- 💖 4 - Well-organized, DRY, easily readable, good variable/function names, would pass linter w/minimal issues
+- 👍 3 - Minor repetition in logic, a couple confusing variable/function names, minor formatting issues
+- 👎 2 - Not idiomatic for language chosen, difficult to understand and trace
+- 🛑 1 - Unable to understand, non-standard formatting, variable names difficult to follow
+
+### Data structures and runtime
+
+Data structures are chosen appropriately for low complexity and low memory allocation.
```

Comment:
The list below seems to be about code organization as well as data structures (e.g. "repetition of the same logic"). Maybe we want a more general description here.